src/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -728,9 +728,7 @@ def evaluate(self, dataset, metric='auto', batch_size=256, verbose=True):
             hclusterSort,
             l2Dist
         )
-
-        evaluation_result = {k: ret[k] for k in ['accuracy', 'f1_score', 'precision',
-                                                 'recall', 'auc', 'roc_curve', 'confusion_matrix']}
+        evaluation_result = {k: ret[k] for k in metrics}
```

Comment:
Are there any metrics we want to exclude? This list of metrics we were filtering to looks like an attempt at a fix for #1974 but I guess that issue is still open anyway.